### PR TITLE
WIP ENS lookup

### PR DESCRIPTION
**Pull request by subramanianv**

### CODE REVIEW

**Comment by subramanianv** [comment]
closes #223

**Comment by subramanianv** [comment]
@villanuevawill The bounties API portion of the PR is ready. Can you do a review of the PR ?
Thanks !

**Comment by villanuevawill** [comment]
@subramanianv this is looking great and an excellent job.

The main point of feedback here is as follows:
You are grabbing the domains they own and just grabbing the root domain from it.
We actually just want to use grab all domains the user owns and only keep it if it is a root domain. (IE. if someone has john.gitcoin.eth we wouldn't count it at all.  We would count it if they own johnthemighty.eth or something like that. Does that make sense as a premise?

File: bounties_api/bounties/settings.py [hunk 1]
```diff
@@ -241,15 +241,15 @@
 GITHUB_TOKEN = os.environ.get('github_token', '')
 SNS_ADDRESS = os.environ.get('sns_address', '')
 
-networks = {
+NETWORKS = {
```
Comment (villanuevawill) [inline bounties_api/bounties/settings.py]: Is this used anywhere else in the application? I don't think it actually is... but just double-checking if you did a search?

Comment (subramanianv) [reply bounties_api/bounties/settings.py]: I did a search and didn't find it anywhere else

**Comment by subramanianv** [comment]
@villanuevawill Thanks for the clarification. I made the required change to save only domains that are root domains. Please take a look.

**Comment by villanuevawill** [comment]
@subramanianv issue with circle credentials. I'm going to close this PR and reopen it to fix :)

**Pull request closed without merging**